app/kubernetes/views/applications/stats/statsController.js

```diff
@@ -149,7 +149,8 @@ class KubernetesApplicationStatsController {
         this.initCharts();
       });
     } catch (err) {
-      this.Notifications.error('Failure', err, 'Unable to retrieve application stats');
+      const errData = err.err.data;
```

Comment:
the problem is that not all errors has the same structure. I advice going to the specfic place where this error is thrown and change the structure there